configs/katello/3.9.yaml

```diff
@@ -0,0 +1,270 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.8.0:
+    :redmine_version_id: 356
+  :3.7.0:
+    :redmine_version_id: 264
+  :3.7.1:
+    :redmine_version_id: 355
+:releases:
+  :3.9.0:
+    :redmine_version_id: 899
+:code_name: New England IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.8
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-installer:
+    :branch: KATELLO-3.8
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/1.19
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :puppet-katello:
+    :branch: 8.0.2
+    :repo: https://github.com/Katello/puppet-katello
+  :hammer-cli-katello:
+    :branch: 0.14.0
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :foreman-packaging:
+    :branch: rpm/1.19
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :katello-host-tools:
+    :branch: 3.3.5
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :bastion:
+    :branch: 6.1.12
+    :repo: https://github.com/Katello/bastion.git
+  :puppet-pulp:
+    :branch: 5.5.0
+    :repo: https://github.com/Katello/puppet-pulp
+  :puppet-candlepin:
+    :branch: 5.1.1
+    :repo: https://github.com/Katello/puppet-candlepin
+  :puppet-certs:
+    :branch: 4.2.0
+    :repo: https://github.com/Katello/puppet-certs
+  :puppet-qpid:
+    :branch: 4.2.2
+    :repo: https://github.com/Katello/puppet-qpid
+  :puppet-foreman_proxy_content:
+    :branch: 7.2.0
+    :repo: https://github.com/Katello/puppet-foreman_proxy_content
+  :puppet-service_wait:
+    :branch: 1.0.4
+    :repo: https://github.com/theforeman/puppet-service_wait.git
+:ignores:
+  - 23322
```

Comment:
Ignores already?

Reply:
I guess I wasn't sure if ignores are kept for each one, but they probably are not, I'll go ahead and take them out.

Reply:
Yea - you can drop ignores